velox/functions/prestosql/tests/ReduceTest.cpp

```diff
@@ -205,3 +205,15 @@ TEST_F(ReduceTest, finalSelectionLargerThanInput) {
 
   assertEqualVectors(makeFlatVector<int64_t>({2, 1}), result);
 }
+
```

Comment:
curious, how is the test case `finalSelection` passing now?

Reply:
Recently there are some PRs for Null handling in functions are merged like 6391. Maybe they introduced changes affecting this case.